oasislmf/preparation/gul_inputs.py

```diff
@@ -537,6 +570,12 @@ def write_gul_input_files(
         if oasis_files_prefixes.get('complex_items'):
             oasis_files_prefixes.pop('complex_items')
 
+    # If no amplification IDs then remove corresponding file name from files
+    # prefixes dict
+    if 'amplification_id' not in gul_inputs_df:
+        if oasis_files_prefixes.get('amplifications'):
```

Comment:
if 'amplification_id' not in gul_inputs_df:
    oasis_files_prefixes.pop('amplifications', None)
will pop the key if present